### ISSUE TITLE
Update toc should not change the undotree

### ISSUE THREAD

**Issue by wsdjeg** [mzlogin/vim-markdown-toc#26]
When use this plugin to edit a md file, after saving file, the toc is updated, but when I type `u` the cursor move to the toc menu, I think the undotree of current file is been updated.

**Comment by wsdjeg** [comment]
Maybe we should use `:undojoin` command, I will look into it to see if I can send a PR.

**Comment by mzlogin** [comment]
Yes, that's a problem. We should not change the undolist when there is no toc change.

I'll look into it. And, welcome to PR for this.

**Comment by wsdjeg** [comment]
I got another issue with this plugin, after save markdown file, `` `` `` in normal mode does not work, we can not jump to the position when last jump, but  will jump to the beginning of TOC, I think it is due to how this plugin works.

I will look into this plugin, and I will fix it before https://github.com/SpaceVim/SpaceVim/pull/806 is merged. 

in this PR I will update SpaceVim's markdown layer.

**Comment by mzlogin** [comment]
Yes, I reproduce it. Maybe we can get help from `keepjumps`. Thanks for your work.
